### PR TITLE
adding service account, role and role binding

### PR DESCRIPTION
**Pull request by sepulworld**

### CODE REVIEW

File: helm/dinghy-ping/templates/rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  name: {{ include "dinghy-ping.fullname" . }}
+automountServiceAccountToken: true
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "dinghy-ping.fullname" . }}-pod-reader
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
```
Comment (kcmartin) [inline helm/dinghy-ping/templates/rbac.yaml]: resources: ["pods", "pods/log"]

File: helm/dinghy-ping/templates/rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "dinghy-ping.fullname" . }}
```
Comment (kcmartin) [inline helm/dinghy-ping/templates/rbac.yaml]: I believe we need namespace here

Comment (kcmartin) [reply helm/dinghy-ping/templates/rbac.yaml]: `namespace: default`
Or, however we're templating it

Comment (sepulworld) [reply helm/dinghy-ping/templates/rbac.yaml]: Looks like `{{ .Release.Namespace }}` will work here cool!

**Pull request merged**